nmap.cc

```diff
@@ -1855,6 +1855,12 @@ int nmap_main(int argc, char *argv[]) {
   Strncpy(mytime, ctime(&timep), sizeof(mytime));
   chomp(mytime);
 
+  char scanninghost[1024];
```

Comment:
this should be 256 bytes https://tools.ietf.org/html/rfc1035#section-2.3.4

Reply:
Good spot. I'll change it.